sdk/python/test_data/components/container_component_with_list_of_artifacts.yaml

```diff
@@ -0,0 +1,46 @@
+# PIPELINE DEFINITION
+# Name: comp-with-artifact-list
+# Inputs:
+#    input_list: system.Artifact
+components:
+  comp-comp-with-artifact-list:
+    executorLabel: exec-comp-with-artifact-list
+    inputDefinitions:
+      artifacts:
```

Comment:
Should we expect to see `is_artifact_list` in the YAML?

Reply:
Yes, updated.